client/src/Components/SearchCountry/SearchCountry.css

```diff
@@ -5,11 +5,10 @@
   z-index: 11;
   height: auto;
   width: auto;
-  padding: 20px;
   font-family: Roboto, sans-serif;
 }
 .SearchCountry .SearchCountry__input {
-  padding: 20px;
-  height: 50px;
+  padding: 10px 20px;
+  height: 30px;
   box-shadow: -3px 3px 2px 2px rgba(0, 0, 0, 0.2);
 }
```

Comment:
I see that you fixed some CSS..was that centering the country maps..or was that only part of the puzzle?